app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -49,9 +49,7 @@ def details
     end
 
     def base_path
-      return "/government/ministers" if I18n.locale == :en
-
-      "/government/ministers.#{I18n.locale}"
+      "/government/ministers"
```

Comment:
I think it would be worth explaining why we only present and render this page in English in the commit note if we know - is that a policy, or in the styleguide or something?

Reply:
Will add this information to the commit message (most roles don't have translations).